internal/apis/certmanager/types_issuer.go

```diff
@@ -292,6 +292,13 @@ type CAIssuer struct {
 	// certificate will be issued with no OCSP servers set. For example, an
 	// OCSP server URL could be "http://ocsp.int-x3.letsencrypt.org".
 	OCSPServers []string
+
+	// The CA Issuers server list is an X.509 v3 extension that defines
+	// a list of URLs of CA Issuers. If not set, the certificate will be
+	// issued with no CA Issuers servers set. For example, a CA Issuers server
+	// URL could be "http://ca.domain.com/ca.crt".
+	// +optional
```

Comment:
suggestion: I think this comment could maybe be a little clearer. I'll use the new name I suggested in the other comment. How does this sound?

```suggestion

	// IssuingCertificateURLs is a list of URLs which this issuer should embed into certificates
	// it creates. See https://www.rfc-editor.org/rfc/rfc5280#section-4.2.2.1 for more details.
	// As an example, such a URL might be "http://ca.domain.com/ca.crt".
	// +optional
```